src/main/java/org/spoken_tutorial/health/elasticsearch/controllers/HomeController.java

```diff
@@ -0,0 +1,157 @@
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HomeController {
+
+    private static final Logger logger = LoggerFactory.getLogger(HomeController.class);
+
+    @Autowired
+    TutorialSearchRepository tutorialRepo;
+
+    @Autowired
+    QueueManagementRepository queRepo;
+
+    @Autowired
+    QueueManagementService queuemntService;
+
+    @Autowired
+    TutorialSearchService tutSearchService;
+
+    private Timestamp getCurrentTime() { // Current Date
+
+        Date date = new Date();
+        long t = date.getTime();
+        Timestamp st = new Timestamp(t);
+
+        return st;
+    }
+
+    private boolean doesFileExist(String filePath) {
+
+        Path path = Paths.get(filePath);
+        return Files.exists(path);
+    }
+
+    @GetMapping("/findAll")
+    public List<QueueManagement> findAll() {
+        return queRepo.findAll();
+    }
+
+    public Map<String, String> addDocument(String documentId, String documentType, String documentPath,
+            String documentUrl, int rank, String view_url, String language, Optional<String> category,
+            Optional<String> topic, Optional<String> outlinePath, String requestType) {
+
+        Map<String, String> resultMap = new HashMap<>();
+        long queueId = queuemntService.getNewId();
+
+        logger.info(
+                "RequestType:{} Language:{} View_URL: {} documentId:{} documentPath:{} documentType:{} outlinePath:{}",
+                requestType, language, view_url, documentId, documentPath, documentType, outlinePath);
+
+        if (documentPath != null && !doesFileExist(documentPath)) {
+            resultMap.put("status", "failed");
+            resultMap.put("reason", "document file does not exist");
+            return resultMap;
+        }
+
+        if (outlinePath != null && outlinePath.get() != null && !doesFileExist(outlinePath.get())) {
+
+            resultMap.put("status", "failed");
+            resultMap.put("reason", "outline file does not exist");
+            return resultMap;
+
+        }
+
+        QueueManagement queuemnt = new QueueManagement();
+        queuemnt.setQueueId(queueId);
+        if (outlinePath != null)
+            queuemnt.setOutlinePth(outlinePath.get());
+        queuemnt.setRequestTime(getCurrentTime());
+        queuemnt.setRequestType(requestType);
+        queuemnt.setDocumentId(documentId);
+        queuemnt.setDocumentType(documentType);
+        queuemnt.setDocumentPath(documentPath);
+        queuemnt.setDocumentUrl(documentUrl);
+        queuemnt.setRank(rank);
+        queuemnt.setView_url(view_url);
+        queuemnt.setLanguage(language);
+        queuemnt.setMessage("success");
```

Comment:
```suggestion
        queuemnt.setStatus("pending");
```
replace witn setStatus of pending